pubpub/platform
# fix: make default title for child pub and form pubtype, not evaluation

- **fix: show pubtype name rather than evaluation in expired layout**
- **fix: show pubtype name rather than evaluation on form fill page**
- **fix: show pubtype name rather than evaluation as default title in pub child table**

## Issue(s) Resolved
Resolves #556 

## Test Plan

1. Create a child pub
2. Should show the pubtype of that pub in the child pub table
3. Create a form for e.g. a submission
4. Go to the fill or expired form
5. See that it says "Submission for" instead of "Evaluation for"

## Screenshots (if applicable)

## Optional

### Notes/Context/Gotchas

### Supporting Docs
